apis/Google.Cloud.Speech.V1Beta1/Google.Cloud.Speech.V1Beta1/RecognitionAudioPartial.cs

```diff
@@ -23,6 +23,8 @@ namespace Google.Cloud.Speech.V1Beta1
 {
     public partial class RecognitionAudio
     {
+        private static readonly HttpClient s_defaultHttpClient = new HttpClient();
```

Comment:
I don't think so - I don't believe constructing an unused HttpClient is terribly expensive,